app/src/main/java/com/sdp/movemeet/Sport.java

```diff
@@ -18,6 +18,6 @@
     Gym,
     Dancing,    
     VolleyBall,
-    Parkour,
-    Tricking
+    Tricking,
```

Comment:
Why they are exchange ?

Reply:
I think its because I deleted those two in a commit by mistake , so I added them again. Nothing really important

Reply:
Yes, you better don't delete them ahah! 💥